wpengine/faustjs
# Add generatePossibleTypes script to `@faustwp/cli`

## Description

These changes port the `generatePossibleTypes.js` script from the example project into the new `@faustwp/cli` package.

## Testing

1. Checkout the `MERL-426-generate-possible-types` branch.
2. From the project root, run `npm run build` in order to build the CLI.
3. Update `NEXT_PUBLIC_WORDPRESS_URL` in the **faustwp** example project to point a new site. 
4. From the project root, run `npm run generate -w @faustwp/next-headless-getting-started` in order to test the new generate command.
5. Notice that `possibleTypes.json` has changed in the **faustwp** example project.

## Screenshots

<img width="578" alt="Screen Shot 2022-10-05 at 4 04 55 PM" src="https://user-images.githubusercontent.com/6676674/194152730-f7b2b018-7702-472d-ae7d-26d946e599be.png">
